.envrc

```diff
@@ -0,0 +1 @@
+use asdf
```

Comment:
Ahhh, I get it now. Interesting! 

Does this work if asdf isn't installed but direnv is?

Reply:
yep that's possible via `has` directive. added a check ;) https://github.com/livebud/bud/pull/393/commits/0bcfd6ccdc598ae3940196406135289653fbcde7

note: the related docs can be found over here: https://direnv.net/man/direnv-stdlib.1.html

Reply:
Neat!